src/zed_char.mli

```diff
@@ -99,6 +137,27 @@ val for_all : (UChar.t -> bool) -> t -> bool
 val iter : (UChar.t -> unit) -> t -> unit
   (** [iter f char] applies [f] on all characters of [char]. *)
 
+(** The prefix 'unsafe_' of [unsafe_of_char] and [unsafe_of_uChar] means
+  the two functions do not check if [char] or [uChar] being transformed
+  is a valid grapheme. There is no 'safe_' version, because the scenario
+  we should deal with a single [char] or [uChar] is when the char
+  sequence are individual, incompelte. For example, when we are reading
```

Comment:
"incomplete" is misspelled here.

Reply:
Thanks, it will be fixed in the 2.0.1 PR.